### DISCUSSION TITLE
SwiftFormat configuration?

### DISCUSSION THREAD

**Discussion by JackYoustra** [pointfreeco/swift-composable-architecture#1464] [Q&A]
On a bit of a lighter note, I've noticed that the included examples aren't formatted like the default four spaces. Does anyone have a configuration file that they've use and like?

**Comment by mbrandonw** [comment]
Hi @JackYoustra, all of our projects are formatted with 2 spaces, which is how we have our local Xcode configured. Further, each commit to `main` causes swift-format to run, which enforces our formatting style.

If you are contributing code to this project there's no need to fret about the styling. Feel free to code in whatever style you want, and it will automatically be reformatted when the code is merged.

**Reply by stephencelis** [reply]
Further, for those that are curious, we just use the default swift-format settings, which are also 2 spaces :smile:
